src/core/gov/msgs/MsgVote.ts

```diff
@@ -13,7 +13,7 @@ export class MsgVote extends JSONSerializable<MsgVote.Data> {
   constructor(
     public proposal_id: number,
     public voter: AccAddress,
-    public option: MsgVote.Option
```

Comment:
`proposal` and `vote` seems receiving enum, but only this one changed to receive number.

Reply:
oh this is actually deleted line.

Reply:
That is deleted line.